tests/pulp-in-one.sh

```diff
@@ -8,7 +8,7 @@ set -o pipefail
 
 mkdir -p settings
 
-PULP_TAG=${PULP_TAG:-"3.21"}
+PULP_TAG=${PULP_TAG:-"3.45"}
```

Comment:
Hmm, do we need to update this now: https://github.com/stackhpc/ansible-collection-pulp/blob/master/.github/workflows/pull_request.yml#L19-L20?